docs/core-concepts/packages.md

```diff
@@ -0,0 +1,170 @@
+  },
+  "frameworks": {
+    "netcoreapp": {}
+  }
+}
+```
+
+Target Frameworks
+=================
+
+.NET Core packages each support a set of target frameworks, declared with target framework folders (within the lib and ref folders mentioned earlier). Target frameworks describe an available API set (and potentially other characteristics) that you can rely on when you target a given framework. They are versioned as new APIs are added.
+
+For example, [System.IO.FileSystem](https://www.nuget.org/packages/System.IO.FileSystem) supports the following target frameworks:
+
+- net46
+- netstandard1.3
+- 6 different Xamarin platforms (for example, xamarinios10)
+
+It is useful to contrast the first two of these target frameworks, since they are examples of the two different ways that target frameworks are defined.
+
+The `net46` framework represents the available APIs in the .NET Framework 4.6. You can produce libraries  compiled with the .NET Framework 4.6 reference assemblies and then distribute those libraries in NuGet packages in a net46 lib folder. It will be used for apps that target the .NET Framework 4.6 or that are compatible with it. This is how all framework targets have traditionally worked.
+
+The `netstandard1.3` framework is a package-based target framework, so it doesn't expose or describe any APIs in the absence of packages that target that framework. It is the interplay of the framework and the packages that define the set of APIs that are available when targeting the framework.
+
+Package-based Target Frameworks
+===============================
+
+There is a two-way relationship between frameworks and packages. The first part is defining the APIs available for a given framework, for example `netstandard1.3`. Continuing with that example, there are no APIs that are inherently part of `netstandard1.3`. All of the `netstandard1.3` APIs come from packages that target `netstandard1.3` or compatible frameworks, such as `netstandard1.0`. These packages and the APIs they expose define the framework.
+
+The second part of the relationship is asset selection. Packages can contain assets for multiple frameworks. Given a reference to a set of packages and/or metapackages, the target framework is needed to determine which asset should be selected, for example `net46` or `netstandard1.3`. It is important to select the correct asset. For example, a `net46` asset is not likely to be compatible with .NET Framework 4.0 or .NET Core 1.0.
+
+It is an interesting question of where a package-based target framework's definition ends and where consumption of that definition starts. There isn't a crisp answer for that. The answer is a function of a given project.json file. Your dependencies create your view of the framework, independent of the publisher(s) of those dependencies. This means that we can consider the definition of `netstandard` to be greater than the libraries that are nominally part of .NET Core or shipped by Microsoft.
```

Comment:
> This means that we can consider the definition of `netstandard` to be greater than the libraries that are nominally part of .NET Core or shipped by Microsoft.

But at the end of the day, aren't I limited by the set of APIs defined to be compatible with a given framework?  I'm not inclined to believe this last section based on what I've read before this.